assets/svelte/components/SelectedElementFloatingMenu/DragMenuOption.svelte

```diff
@@ -243,16 +243,34 @@
     let i = 0
     if (dragDirection === "vertical") {
       while (i < newInfos.length && i < newIndex) {
-        distance += newInfos[i].height + newInfos[i].marginTop + newInfos[i].marginBottom
+        let gap = 0
```

Comment:
Instead of assuming the "void" between elements comes determined by the margin (which is plainly wrong in anything but the most basic layout techniques), we measure the distance from the rightmost edge of one elements and the leftmost edge of the next.

That is a generic solution that works for marging, flebox, css-grid and probably any other layout approach.